viewer/src/data/model/ConsumedSector.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { LevelOfDetail } from './LevelOfDetail';
+import { SectorMetadata } from '../../models/cad/types';
+
+export interface ConsumedSector {
```

Comment:
As mentioned in our chat for reference to others: I prefer if we can leverage the type system to avoid having incomplete data passed to functions, which again means we do not need to perform sanity checks everywhere. By having one type, we could easily pass said type without fetched data to the parser, and without parsed data to the Three.js consumer. I suggest we stick with different types to avoid such errors.